components/Header/index.js

```diff
@@ -20,7 +20,7 @@ const Header = ({
           </h2>
           <p className="mt-2 text-gray-500 text-xl md:text-2xl">{caption}</p>
           {buttonCaption && (
-            <Button caption={buttonCaption} link="/categories" />
+            <Button caption={buttonCaption} link={"#categories"} />
```

Comment:
This was a `/` before Negin. Just wanted to check it still works like this.

Reply:
@ARTurnerGit  The reason for this is we thought since this is already being shown on the main page, it makes sense to jump to categories on the page instead of showing a new page with exact same information, that's why it's been changed.

Reply:
Of course, it's a fragment, I probably should have guessed that. Fair enough - I don't know what the pages look like so sometimes it's tricky to work out things like that.